libse/StringExtensions.cs

```diff
@@ -185,5 +186,17 @@ public static string RemoveControlCharactersButWhiteSpace(this string s)
             return sb.ToString();
         }
 
+        public static string CapitalizeFirstLetter(this string s, CultureInfo ci = null)
+        {
+            var si = new StringInfo(s);
+            if (ci == null)
+                ci = CultureInfo.CurrentCulture;
+            if (si.LengthInTextElements > 0)
+                s = si.SubstringByTextElements(0, 1).ToUpper(ci);
+            if (si.LengthInTextElements > 1)
+                s += si.SubstringByTextElements(1);
```

Comment:
No, this should only capitalize the first letter (or leave it unchanged if it cannot be capitalized).
For example (comboBox item),
 `português (Portugal)` ==> `Português (Portugal)`
not,
 `português (Portugal)` ==> `Português (portugal)`


The functionality that you desire, should be provided by a different method.
Or use `s.ToLower(ci).CapitalizeFirstLetter(ci)`—as in [*interjections* commit](https://github.com/xylographe/subtitleedit/commit/a15d79cce920e6bade42a9f2f91b857ea9a883c9#diff-f42968ff2e33bbfd08f6da5a14fb1acdR711).